RELEASE.md

```diff
@@ -225,10 +216,9 @@ git checkout v$VERSION
 > architectures on **one** platform, and then copy it to the rest of the
 > platforms, before creating our .deb / .rpm / .msi / app bundles.
 
-## macOS release
-
-To make a macOS release, go to macOS build machine:
+### macOS Release
 
+#### First Time Signing Machine Setup
```

Comment:
I'd change this to "Initial Setup" or "First-time Setup". Same for the rest of the similar sections.

Reply:
Looks good, feel free to merge.